db/migrate/20150306060237_unread_migration.rb

```diff
@@ -0,0 +1,16 @@
+class UnreadMigration < ActiveRecord::Migration
+  def self.up
+    create_table :read_marks, force: true do |t|
+      t.integer :readable_id, foreign_key: false
+      t.integer :user_id, null: false
+      t.string :readable_type, null: false, limit: 20
```

Comment:
https://github.com/ledermann/unread/commit/b90e975654a0f2fe34ea20df5ed129749de8b310
They removed limit 20 in the latest commit, I think we'd better remove this also.